pyproject.toml

```diff
@@ -157,6 +157,7 @@ ptpython = "3.0.23"
 ptvsd = "4.3.2"
 pytest-cov = "4.1.0"
 watchdog = "3.0.0"
+ruff = "0.0.289"
```

Comment:
Seems like a good idea.  The dependencies seem to be in alphabetical order though, so it probably needs to be moved up a line. :wink:

Reply:
I think the alphabetical order is a holdover from `requirements.txt` formatting. Personally I don't care whether it's alphabetical or not /shrug

Reply:
No worries then. :smile: